packages/@cruise-automation/button/package.json

```diff
@@ -10,14 +10,13 @@
     "watch": "NODE_ENV=development npm run build-dev --watch"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "styled-components": "^3.3.0"
```

Comment:
Got rid of styled-components in this package as it conflicted with the other styled-components in the root and couldn't easily figure out how to resolve it.